api/src/pcapi/connectors/entreprise/backends/api_entreprise.py

```diff
@@ -242,7 +242,7 @@ def get_siren_open_data(self, siren: str, with_address: bool = True) -> models.S
             siren=siren_data["siren"],
             name=self._get_name_from_sirene_data(siren_data),
             head_office_siret=head_office_siret,
-            ape_code=siren_data["activite_principale"]["code"],
+            ape_code=self._format_ape_code(siren_data["activite_principale"].get("code", None)),
```

Comment:
Détail : inutile de préciser `None`, valeur par défaut.
```suggestion
            ape_code=self._format_ape_code(siren_data["activite_principale"].get("code")),
```
(idem plus loin)